test/gracefulstop_test.go

```diff
@@ -216,3 +217,89 @@ func (s) TestGracefulStopClosesConnAfterLastStream(t *testing.T) {
 		<-gracefulStopDone // Wait for GracefulStop to return.
 	})
 }
+
+func (s) TestGracefulStopBlocksUntilGRPCConnectionsTerminate(t *testing.T) {
+	// This tests ensures that GracefulStop() blocks until all ongoing
+	// client GRPC calls finished.
```

Comment:
Nit: can you move to above the function definition. also switch "client GRPC calls" to "rpcs" please.

Reply:
Ok, I'll also do it for the existing testcases in this file then.

Reply:
done